include/veriblock/blockchain/blocktree.hpp

```diff
@@ -109,10 +109,32 @@ struct BlockTree {
     return acceptBlock(block, state, true);
   }
 
+  void invalidateBlockByHash(const hash_t& blockHash) {
+    index_t* blockIndex = getBlockIndex(blockHash);
```

Comment:
here you should check that block index is not nullptr.
`getBlockIndex` returns nullptr when it can't find a block with this hash